extension/src/Runner/runnerResultStream.ts

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import * as events from "events";
+import { Readable } from "stream";
+
+export class RunnerResultStream extends events.EventEmitter implements IRunnerResultStreamEvent {
+    constructor(public readonly stderr: Readable, public readonly stdout: Readable) {
+        super();
+    }
+}
+
+interface IRunnerResultStreamEvent {

```

Comment:
There are already events defined in the stderr and stdout streams. Does this mean those are not enough?

Reply:
yes, i'd like to mark when the running completes and when some errors occurred to the runner itself as well